src/platform/forms-system/src/js/utilities/save-in-progress-messages.js

```diff
@@ -0,0 +1,19 @@
+import { APP_TYPE_DEFAULT } from '../constants';
+
+const appType = formConfig =>
+  formConfig.customText?.appType || APP_TYPE_DEFAULT;
+
+export function inProgressMessage(formConfig) {
+  const defaultMessage = `Your ${appType} is in progress.`;
+  return formConfig.saveInProgress?.inProgress || defaultMessage;
+}
+
+export function expiredMessage(formConfig) {
+  const defaultMessage = `Your saved ${appType} has expired. If you want to apply for benefits, please start a new ${appType}.`;
+  return formConfig.saveInProgress?.expired || defaultMessage;
+}
+
+export function savedMessage(formConfig) {
+  const defaultMessage = `Your ${appType} has been saved.`;
```

Comment:
The default message no longer includes the form title, but I think that's fine since we give teams to choose their own message.

https://github.com/department-of-veterans-affairs/vets-website/pull/14468/files#diff-8a032b6554324e40d103495d986dc65cL61

 ```jsx
    Your {formTitles[formId]} {appType} has been saved.
 ```

Reply:
Yeah, exactly. If we like this approach, I'll reach out to Peggy and see if she'd like to make any changes to these default messages.